README.md

```diff
@@ -11,27 +11,30 @@ The <b>kamon-akka</b> module require you to start your application using the Asp
 
 ### Getting Started
 
-Kamon akka module is currently available for Scala 2.10, 2.11 and 2.12.
+Kamon akka module is currently available for Scala 2.10, 2.11, 2.12 and 2.13.
+
+**Since Kamon 2.0 there are no separate artifacts per akka version, the jar is released as kamon-akka**
 
 Supported releases and dependencies are shown below.
 
 | Artifact              | status | jdk        | scala            | akka   |
 |:---------------------:|:------:|:----------:|------------------|:------:|
 |  kamon-akka-2.4 1.1.0 | stable | 1.7+, 1.8+ | 2.11, 2.12       | 2.4.x  |
 |  kamon-akka-2.5 1.1.0 | stable | 1.8+       | 2.11, 2.12       | 2.5.x  |
+|  kamon-akka 2.0.1     | stable | 1.8+       | 2.11, 2.12, 2.13 | 2.6.x  |
```

Comment:
Hey @DanieleSassoli, a small edit here: the very same dependency supports Akka 2.4 (with limitations), 2.5 and 2.6, and we are not actively developing the 1.x series anymore.

Could you please mention only the Kamon 2.x version with all supported Akka versions?